frontend/package.json

```diff
@@ -55,6 +55,7 @@
     "history": "4.9.0",
     "konva": "4.0.9",
     "lodash": "4.17.15",
+    "markdown-linkify": "^1.0.1",
```

Comment:
lock down the version please

We've had plenty of troubles in the past where a later version of the library changed the API and randomly broke deployments 😢